src/components/FishCard.js

```diff
@@ -65,14 +71,34 @@ const FishName = styled.h2`
   padding: 10px;
 `;
 
-const FishNameLatin = styled.div`
-  list-style: none;
+const ContainerFishProperties = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+
+  align-content: space-evenly;
   font-family: Arial, Helvetica, sans-serif;
   color: #28382c;
   margin-top: 0.1em;
```

Comment:
You could use the shorthand here for the sake of clarity: 
`margin: 0.1em 20px 10px 34px`